### ISSUE TITLE
Unable to read cookies set in a previous session

### ISSUE THREAD

**Issue by isaiahstjohn** [Mohamed-512/Extra-Streamlit-Components#5]
Here's a simple app to demonstrate the issue.

```python
import streamlit as st
import extra_streamlit_components as stx

st.write("Demo")
          
cookie_manager = stx.CookieManager()

def save_cookie():
    cookie_manager.set("c", "my-cookie")
    
c = cookie_manager.get("c")
st.write(c)
   
if not c:
    st.write('No cookie "c" found')
    st.button("Save cookie", on_click=save_cookie)
```

After clicking the button to set the cookie and then refreshing the page, the app is unable to load the cookie. Yet using the developer console, we can verify that the cookie is there.

**Comment by Mohamed-512** [comment]
May I know what version of the package you are on?

**Comment by isaiahstjohn** [comment]
I'm on version 0.89.0.

**Comment by Mohamed-512** [comment]
Thankd for the effort. Will give it a look in the next few days.

**Comment by Mohamed-512** [comment]
Issue fixed on version 0.1.44.
